src/pyg4ometry/gui/GeometryModel.py

```diff
@@ -22,36 +23,36 @@ def __init__(self):
         self.vtkDict = {}
 
     def createNewRegistry(self, name, type):
-        reg = pyg4ometry.geant4.Registry()
+        reg = _g4.Registry()
         self.registryDict[name] = reg
 
     def loadNewRegistry(self, fileName):
         name = nameFromPath(fileName)
         type = extensionFromPath(fileName)
 
         if type == "gdml":
-            r = pyg4ometry.gdml.Reader(fileName)
+            r = _gdml.Reader(fileName)
             self.registryDict[name] = r.getRegistry()
             self.worldLogicalDict[name] = self.registryDict[name].getWorldVolume()
         elif type == "step" or type == "stp":
-            r = pyg4ometry.freecad.Reader(fileName)
+            r = _freecad.Reader(fileName)
             r.relabelModel()
             r.convertFlat()
             self.registryDict[name] = r.getRegistry()
             self.worldLogicalDict[name] = self.registryDict[name].getWorldVolume()
         elif type == "stl":
-            r = pyg4ometry.stl.Reader(fileName)
-            reg = pyg4ometry.geant4.Registry()
+            r = _stl.Reader(fileName)
+            reg = _g4.Registry()
             log = r.logicalVolume(name, "G4_Galactic", reg)
             self.registryDict[name] = reg
             self.worldLogicalDict[name] = log
         elif type == "inp":
-            r = pyg4ometry.fluka.Reader(fileName)
+            r = _luka.Reader(fileName)
```

Comment:
This should be `_fluka`?

Reply:
Yup.... I'm increasing the test coverage now, that will find many issues. Currently there is a lot untested code as it is rarely used. I'll fix and it will come with my next MR